packages/strapi-hook-mongoose/lib/utils/index.js

```diff
@@ -5,7 +5,8 @@
  */
 
 module.exports = mongoose => {
-  require('mongoose-float').loadType(mongoose);
+  var Decimal = require('mongoose-float').loadType(mongoose, 2);
```

Comment:
Why `var` while the most uses `const` etc?

Reply:
Due to the 'use strict' tag which enforces ES5 therefore `const` and `let` cannot be used.

Reply:
Really? we do use const/let everywhere in the app.

Reply:
I don’t mind using them but I’m working off whatever code was there initially.